src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -159,12 +159,12 @@ public class TxPool : ITxPool, IDisposable
         public IDictionary<Address, Transaction[]> GetPendingTransactionsBySender() =>
             _transactions.GetBucketSnapshot();
 
-        public IDictionary<Address, Transaction[]> GetPendingLightBlobTransactionsBySender() =>
-            _blobTransactions.GetBucketSnapshot();
-
         public Transaction[] GetPendingTransactionsBySender(Address address) =>
             _transactions.GetBucketSnapshot(address);
 
+        public IDictionary<Address, Transaction[]> GetPendingLightBlobTransactionsBySender() =>
```

Comment:
why move?

Reply:
merge conflict and I ended up with two copies of the same method, I removed the first one and left the duplicate by mistake